awswrangler/s3/_write_parquet.py

```diff
@@ -995,9 +994,9 @@ def store_parquet_metadata(  # pylint: disable=too-many-arguments,too-many-local
         s3_additional_kwargs=s3_additional_kwargs,
         boto3_session=boto3_session,
     )
-    _logger.debug("columns_types: %s", columns_types)
-    _logger.debug("partitions_types: %s", partitions_types)
-    _logger.debug("partitions_values: %s", partitions_values)
+    _logger.debug("Resolved columns_types: %s", columns_types)
+    _logger.debug("Resolved partitions_types: %s", partitions_types)
+    _logger.debug("Resolved partitions_values: %s", partitions_values)
```

Comment:
Could these lists be huge? Should we print only first x or just their lengths? Also, in `_write_text` similar logging for partitions types is removed so not sure maybe we want to get rid of them entirely.

Reply:
`column_types` and `partition_types` shouldn't be as they refer to the columns only
`partitions_values` might be so can change it to log the length instead